### PR TITLE
PFA-4: Add firebase to Android

### PR DESCRIPTION
**Pull request by ductamisme**
"PFA-4: Add firebase to Android"

## Description & Motivation

I'm almost done adding firebase

## Ticket number and links

https://console.firebase.google.com/project/up-personal-finance/overview

## Screenshots

I don't have

## Checklist

- [ Ok] I have reviewed my own code
- [ Ok] I have verified that my commits are properly labelled
- [ Ok] I have updated any related documentation
- [ Ok] I have added appropriate tests for my new changes
- [ Ok] I have updated any existing tests related to my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Personal finance app
 - Dependency injection: Hilt
 - Jetpack Compose + XML
 - Kotlin Flow + LiveData
-- ktlint
+- kotlin
```
Comment (linhvnguyen9) [inline README.md]:
```suggestion
- ktlint
```
ktlint là 1 tool e :)))) https://github.com/pinterest/ktlint

**Review by linhvnguyen9 — APPROVED** [review]
undo lại đoạn thay đổi README là ok r đó e

File: README.md (hunk 1 above)
Comment (ductamisme) [reply README.md]: Dạ vâng ạ :))))

**Review by DawnHNguyen — APPROVED** [review]

**Review by lenhathuy9281 — APPROVED** [review]
a approve mấy đoạn code này 3 lần r mà nhỉ?

**Comment by ductamisme** [comment]
giờ mới là final a ạ :))


Vào Th 5, 16 thg 2, 2023 vào lúc 16:14 Le Nhat Huy <
***@***.***> đã viết:

**Comment by ductamisme** [comment]
chore: [PFA-4] Add firebase to Android

[PFA-4]: https://finance-aibles.atlassian.net/browse/PFA-4?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Pull request merged**